homeassistant/components/logi_circle/strings.json

```diff
@@ -11,16 +11,13 @@
         "description": "Please follow the link below and **Accept** access to your Logi Circle account, then come back and press **Submit** below.\n\n[Link]({authorization_url})"
       }
     },
-    "create_entry": {
-      "default": "Successfully authenticated with Logi Circle."
-    },
```

Comment:
Am I missing something or is this not used.

Reply:
I guess this is used by any helper around.
Core dev might confirm (or not 😋 )

Reply:
@scheric Yep, AFAIK not used. I think at the time, I noticed `create_entry.default` on other integrations I was using as a reference implementation where it also didn't appear to be used, so assumed there may be some flow I'm unaware of that called it.